examples/complete/main.tf

```diff
@@ -51,22 +63,20 @@ module "complete" {
   }
 
   ## security group: Additional rules
-  security_group_rules = {
-    ingress_http = {
+  security_group_ingress = [
+    {
       from_port   = 80
       to_port     = 80
       protocol    = "tcp"
       cidr_blocks = ["0.0.0.0/0"]
```

Comment:
It makes sense to restrict the SG cidr to the VPC just for example sake, `0.0.0.0/0` is a unwanted pattern